documentation/modules/ROOT/pages/operations/debezium-server.adoc

```diff
@@ -478,6 +478,48 @@ The configuration will need to contain the list of predicates, implementation cl
 
 |===
 
+[id="debezium-async-engine-properties"]
+=== Asynchronous Engine Properties
+
+By default, Debezium server uses `AsyncEmbeddedEngine` as the processing engine.
+You can configure following options for asynchronous embedded engine:
+
+[cols="35%a,10%a,55%a"]
+|===
+|Property
+|Default
+|Description
+
+|`record.processing.threads`
+|The number of available cores.
+|The number of threads to use for processing CDC records.
+If you want to use all available cores on given machine, you can use the `AVAILABLE_CORES` placeholder.
+If the value is not specified, threads are created as needed, using Java https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/concurrent/Executors.html#newCachedThreadPool()[cached thread pool].
+
+|`record.processing.shutdown.timeout.ms`
+|1000
+|Maximum time in milliseconds to wait for processing submitted records after a task shutdown is called.
```

Comment:
```suggestion
|Maximum time, in milliseconds, that the engine waits to process pending records after a task shutdown is called.
```
Does this represent the interval before the engine waits before it begins to process pending records, or is it the interval the determines the amount of time allocated to complete processing after a shutdown is called?

Reply:
It's the latter @roldanbob